### DISCUSSION TITLE
Are you using Control Center? Let us know!

### DISCUSSION THREAD

**Discussion by blt950** [Vatsim-Scandinavia/controlcenter#383] [General]
We'd like to know who uses Control Center, this helps us to understand the target group when creating features, but also know which changes might affect whom.

Please reply in this thread with your division/subdivision and optionally a link to your installation of Control Center 😄

**Comment by blt950** [comment]
VATSIM Scandinavia uses it obviously, https://cc.vatsim-scandinavia.org/ 😄

**Comment by amosngSP** [comment]
VCLvACC is using it too!
https://cc.vclvacc.net/

**Comment by sarantos92** [comment]
we dont use it yet as i am the only one who has some knowledge to run it, but my time is limited
